pkg/NAMESPACE

```diff
@@ -181,5 +180,12 @@ export("cacheTable",
 
 export("sparkRSQL.init",
        "sparkRHive.init")
-export("print.structType",
-       "print.structField")
+
+export("structField",
+       "structField.jobj",
```

Comment:
Just curious: does the `structField` constructor fail if we don't export `structField.jobj` ?